bundles/pom.xml

```diff
@@ -2,681 +2,2011 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 
+
```

Comment:
Somehow this PR introdeuced many whitelines, can you revert and only add this binding.